Dockerfile

```diff
@@ -1,44 +1,55 @@
-ARG dialect
+ARG dialect=sqlite
+
+FROM python:slim as poetry
+
+ARG dialect=sqlite
+
+WORKDIR /root
+RUN pip install poetry
```

Comment:
Please, use [recommended way to install the poetry](https://python-poetry.org/docs/master/#installation).

Reply:
No. It requires curl. curl = more dependencies. More dependencies = larger docker image.

Reply:
So... squash it? `RUN apt-get install curl -y && curl .../install_poetry.py && apt-get remove curl -y`, even without it, curl will not go to final step, because you use `COPY --from`.

Reply:
curl install method recommended when you have multiple python versions. In docker we have only one python version, so this method is faster, than using a curl method.

Reply:
Anyway I changed it in https://github.com/PerchunPak/pinger-bot/pull/51/commits/ea7dfae3200652e2a3f24e75e5c394fb43a9b4a2.